var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -42,7 +42,8 @@ class Dealii(CMakePackage):
     version('8.3.0', 'fc6cdcb16309ef4bea338a4f014de6fa')
     version('8.2.1', '71c728dbec14f371297cd405776ccf08')
     version('8.1.0', 'aa8fadc2ce5eb674f44f997461bf668d')
-    version('develop', git='https://github.com/dealii/dealii.git', tag='master')
+    version('develop', git='https://github.com/dealii/dealii.git',
+            branch='master')
```

Comment:
Up to you. If you're curious, the line that exempts it is:
https://github.com/LLNL/spack/blob/develop/lib/spack/spack/cmd/flake8.py#L59

We exempt several other directives as well.